### PR TITLE
[GP5-27] Order - As an admin, I want to see all user orders

### PR DESCRIPTION
**Pull request by doomsdoge**

### CODE REVIEW

**Review by haikal1223 — APPROVED** [review]

**Review by BarsMoxxi — APPROVED** [review]

**Review by rohmatsetiawanmta — COMMENTED** [review]
Update uid ke id di file-file berikut
projects\client\src\pages\Admin\Order\index.js
projects\server\controller\adminController.js
projects\server\sequelize\models\cart.js
projects\server\sequelize\models\order.js
projects\server\sequelize\models\user.js
projects\server\sequelize\models\wh_admin.js

setelah itu, cek function fetchWarehouse dan controller fetchWarehouse apakah berjalan normal

**Comment by doomsdoge** [comment]
> 

done

File: projects/client/src/pages/Home/index.js [hunk 1]
```diff
@@ -115,8 +115,9 @@ export default function Home(props) {
     }
   };
   const prevPageHandler = () => {
-    if (page > 1) {
-      setPage(page - 1);
+        if (page > 1) {
+            setPage(page - 1);
+        }
     }
   };
```
Comment (rohmatsetiawanmta) [inline projects/client/src/pages/Home/index.js:122]: delete line 122

**Review by rohmatsetiawanmta — APPROVED** [review]
Ada berlebih satu kurung kurawal tutup aja di projects/client/src/pages/Home/index.js (line 122)

File: projects/client/src/pages/Home/index.js (hunk 1 above)
Comment (doomsdoge) [reply projects/client/src/pages/Home/index.js]: removed

**Review by RaihanAdity — APPROVED** [review]

**Pull request merged**